tcf_website/views/search.py

```
@@ -331,3 +352,31 @@ def apply_filters(results, filters):
         results = results.filter(coursegrade__average__gte=float(min_gpa))
 
     return results.distinct()
+
+
+@api_view(["GET"])
+def autocomplete(request):
+    """implement autocomplete for search bar"""
+    query = request.GET.get("q", "").strip()
+    mode, is_club = parse_mode(request)
```

Ruff (0.14.10)

Unpacked variable `mode` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)